src/signatureUtils.js

```diff
@@ -99,7 +118,15 @@ async function fetchKeyFromApiAndPutInCache (pubKeyUrl, state, publicKeyFileName
  */
 async function getKeyFromCache (state, publicKeyFileNameAsKey) {
   try {
-    return await state.get(publicKeyFileNameAsKey)
+    const keyObj = await state.get(publicKeyFileNameAsKey)
+    /**
+     * key json obj fetched from cache in below format
+     * {
+     *     "value":"pem public key",
+     *     "expiration":"24h timestamp"
```

Comment:
Just wondering if `expiration` is used anywhere?

Reply:
yes, if nothing provided `aio-lib-state` uses an `expiration` value of `24h` set from the current timestamp. see https://github.com/adobe/aio-lib-state#use